site/content/docs/5.3/examples/form/form.js

```diff
@@ -16,6 +16,19 @@
         // Focus on first error
         const invalidItems = form.querySelectorAll(':invalid')
         invalidItems[0].focus()
+        // Add the id of the corresponding invalid message to each invalid field
+        invalidItems.forEach(element => {
+          const linkedLabel = element.getAttribute('aria-labelledby')
+          const closestInvalidFeedback = element.closest('.mb-3').querySelector('.invalid-feedback').getAttribute('id')
+          const valuesArray = [linkedLabel, closestInvalidFeedback].join(' ')
+          element.setAttribute('aria-labelledby', valuesArray)
+        })
+        // Remove the id of the corresponding invalid message to each valid field
+        const validItems = form.querySelectorAll(':valid:not([type=\'submit\'])')
```

Comment:
Ok for the double quotes.
Not ok for the `formvalidate=novalidate` attribute because it will override the `novalidate` attribute on the `form` element. It will be weird to override something we recommend in the doc.